src/WebJobs.Script.WebHost/Global.asax.cs

```diff
@@ -45,6 +47,25 @@ private static void VerifyAndEnableShadowCopy(WebHostSettings webHostSettings)
             }
         }
 
+        private static void ConfigureMinimumThreads(bool isDynamicSku)
+        {
+            // For information on MinThreads, see:
+            // https://docs.microsoft.com/en-us/dotnet/api/system.threading.threadpool.setminthreads?view=netframework-4.7
+            // https://docs.microsoft.com/en-us/azure/redis-cache/cache-faq#important-details-about-threadpool-growth
+            // https://blogs.msdn.microsoft.com/perfworld/2010/01/13/how-can-i-improve-the-performance-of-asp-net-by-adjusting-the-clr-thread-throttling-properties/
+            //
+            // This behavior can be overridden by using the "ComPlus_ThreadPool_ForceMinWorkerThreads" environment variable (honored by the .NET threadpool).
+
+            // The dynamic plan has some limits that mean that a given instance is using effectively a single core, so we should not use Environment.Processor count in this case.
+            var effectiveCores = isDynamicSku ? 1 : Environment.ProcessorCount;
```

Comment:
Sounds good. I had tested with similar code and gotten the expected result, so this is a bit concerning, but not related to this changes.